### PR TITLE
Just a little housekeeping.

### PR DESCRIPTION
**Pull request by bahoo**
**Untested** because I knew I could author the change quicker than I could get the dev environment setup. :D

Modify that regex pattern as needed.

### CODE REVIEW

File: lib/admin/requests/service.php [hunk 1]
```diff
@@ -86,6 +86,9 @@ public function createRequest($request) {
       'post_content' => $request[self::POST_CONTENT_KEY],
       'post_type' => self::POST_TYPE
     );
+    if(preg_match("/4chan\.org/i", $request['referer'])){
```
Comment (atticoos) [inline lib/admin/requests/service.php]: Can I be picky and ask the regex be a constant?

**Comment by atticoos** [comment]
Haha hellll yeah :smiley:

File: front-page.php [hunk 2]
```diff
@@ -114,6 +114,7 @@
           </label>
         </div>
       </div>
+      <input type="hidden" name="referer" value="<?php echo $_SERVER['HTTP_REFERER']; ?>" />
```
Comment (atticoos) [inline front-page.php:117]: Do you know if the browser will carry over the `HTTP_REFERRER` through the `XMLHttpRequest`? I'm wondering if we can hide this away by checking this server side in the request handler

File: lib/admin/requests/service.php (hunk 1 above)
Comment (atticoos) [inline lib/admin/requests/service.php]: PS don't bother with the other string literals being constants, only the regex

File: front-page.php (hunk 2 above)
Comment (bahoo) [inline front-page.php:117]: It'd probably be the page itself, at that point.

File: lib/admin/requests/service.php (hunk 1 above)
Comment (bahoo) [inline lib/admin/requests/service.php]: Of course... update en route.

File: front-page.php (hunk 2 above)
Comment (atticoos) [inline front-page.php:117]: Yeahh that's what I'm thinking too. I might expand on this later and throw this in a cookie

**Comment by atticoos** [comment]
Will merge after giving it a run through tomorrow. Everything looks good and looks like it'll run just fine. :+1: thanks!

**Pull request merged**